<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FaceDataEditorWidget</class>
 <widget class="QWidget" name="FaceDataEditorWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>528</width>
    <height>489</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QFrame" name="editBoundaryConditionFrame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_7">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Properties</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QtTreePropertyBrowser" name="boundaryConditionPropertyBrowser" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>150</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="faceSelectionFrame">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <item>
            <widget class="QToolButton" name="selectFacesButton">
             <property name="toolTip">
              <string>Select the faces to apply the face data to</string>
             </property>
             <property name="text">
              <string>Select faces</string>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
             <property name="checked">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="selectFromListButton">
             <property name="toolTip">
              <string>Select faces from list.</string>
             </property>
             <property name="text">
              <string>...</string>
             </property>
             <property name="icon">
              <iconset resource="../../resources/SolverSetup.qrc">
               <normaloff>:/contourModeling/icons/list.png</normaloff>:/contourModeling/icons/list.png</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>32</width>
               <height>32</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="faceSelectionSettingsButton">
             <property name="toolTip">
              <string>Show advanced options</string>
             </property>
             <property name="text">
              <string>...</string>
             </property>
             <property name="icon">
              <iconset resource="../../resources/SolverSetup.qrc">
               <normaloff>:/contourModeling/icons/settings.png</normaloff>:/contourModeling/icons/settings.png</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>16</width>
               <height>16</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_3">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QCheckBox" name="applyToAllWallsCheckBox">
             <property name="toolTip">
              <string>Apply the face data to all walls</string>
             </property>
             <property name="text">
              <string>Apply to all walls</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QTableView" name="faceListTable">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>150</height>
            </size>
           </property>
           <property name="editTriggers">
            <set>QAbstractItemView::NoEditTriggers</set>
           </property>
           <property name="selectionBehavior">
            <enum>QAbstractItemView::SelectRows</enum>
           </property>
           <property name="sortingEnabled">
            <bool>true</bool>
           </property>
           <attribute name="horizontalHeaderStretchLastSection">
            <bool>true</bool>
           </attribute>
           <attribute name="verticalHeaderVisible">
            <bool>false</bool>
           </attribute>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QtTreePropertyBrowser</class>
   <extends>QWidget</extends>
   <header location="global">QtTreePropertyBrowser</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>selectFacesButton</tabstop>
  <tabstop>faceSelectionSettingsButton</tabstop>
  <tabstop>applyToAllWallsCheckBox</tabstop>
  <tabstop>faceListTable</tabstop>
 </tabstops>
 <resources>
  <include location="../../resources/SolverSetup.qrc"/>
 </resources>
 <connections/>
</ui>
